ports/raspberrypi/common-hal/wifi/Radio.c

```diff
@@ -161,7 +161,15 @@ void common_hal_wifi_radio_stop_station(wifi_radio_obj_t *self) {
     // (by tcpip_link_status). However since ap disconnection isn't working
     // either, this is not an issue.
     cyw43_wifi_leave(&cyw43_state, CYW43_ITF_AP);
-
+    size_t timeout_ms = (size_t)MICROPY_FLOAT_C_FUN(ceil)(500);
```

Comment:
I think just `500` would be OK? Are you getting a compile error otherwise??
```suggestion
    const size_t timeout_ms = 500;
```

Reply:
The timeout only exists as a last resort, so that the board doesn't hang.
500 as a value is just what I feel is best.

Doing `= 500` is fine, my brain just passively copy-pasted without thinking.